kybra/compiler/kybra_post_install/build.rs

```diff
@@ -0,0 +1,154 @@
+use fs_extra::dir::{copy, CopyOptions};
+use std::path::Path;
+use std::process::Command;
+use tempfile::tempdir;
+
+static ERROR_PREFIX: &str = "Kybra Post Install Build Error";
+
+fn main() -> Result<(), String> {
+    let ic_build = get_and_set_ic_build();
+
+    if ic_build == false {
+        return Ok(());
+    }
+
+    let kybra_version = get_kybra_version()?;
+    let home_dir = dirs::home_dir().ok_or(create_error_string("Home directory not found"))?;
+
+    let python_stdlib_binary_path =
+        home_dir.join(format!(".config/kybra/{kybra_version}/bin/python_stdlib"));
+    handle_python_stdlib_exists(&python_stdlib_binary_path);
+
+    let python_stdlib_src_path = home_dir.join(format!(".config/kybra/{kybra_version}/Lib"));
+    let kybra_post_install_src_lib_path = Path::new("src/Lib");
+
+    if kybra_post_install_src_lib_path.exists() {
+        clean_kybra_post_install_src_lib(&kybra_post_install_src_lib_path)?;
+        return Ok(());
+    }
+
+    copy_from_python_stdlib_src_to_kybra_post_install_src(
+        &python_stdlib_src_path,
+        &kybra_post_install_src_lib_path,
+    )?;
+
+    let github_repo = "https://github.com/RustPython/RustPython.git";
+    let commit_hash = "f12875027ce425297c07cbccb9be77514ed46157";
+
+    let temp_dir = tempdir().map_err(|e| error_to_string(&e))?;
+    git_clone(github_repo, &temp_dir)?;
```

Comment:
So deploying will require an active internet connection? And, it will re-clone the repo every time you deploy? If that's true then ideally we would cache this so they can work offline.